src/main/java/io/github/aquerr/steamwebapiclient/SteamEconServiceWebApiClient.java

```diff
@@ -25,4 +27,16 @@ public TradeHistoryResponse getTradeHistory(TradeHistoryRequest request) {
                 TradeHistoryResponse.class
         );
     }
+
+    /**
+     * Gets a specific trade offer with the WebAPI key.
+     */
+    public TradeOfferResponse getTradeOffer(TradeOfferRequest request) {
```

Comment:
Add `throws ClientException` part here, and information about thrown exception in javadoc. (You can copy it from other methods)